### ISSUE TITLE
Basic testing for i18n-check

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/i18n-check#2]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/i18n-check-action/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](.github/CODE_OF_CONDUCT.md)

### Description

Once we have the scripts migrated over from activist, the next step would be to add pytest based testing to the directory. We’d want to have some fake data in the testing files with valid and invalid i18n uses and make sure that future changes maintains the current functionality. This will also allow us to guide development in the future when we try to expand the functionality of the package. 

### Contribution

Happy to work on this and review when a PR is up! 😊

**Comment by andrewtavis** [comment]
As a part of this, we'd want to use the new YAML file for all check scripts:
- Load in `.i18n-check.yaml`
- Replace all instances of `file_types_to_check`, `directories_to_skip` and `files_to_skip` with the data from the yaml file

Then we can move on to writing Pytest tests for this :)

**Comment by andrewtavis** [comment]
@OmarAI2003, don't forget to write in here so I can assign :)

**Comment by OmarAI2003** [comment]
Here and ready to be assigned! 👍

**Comment by andrewtavis** [comment]
Assigned! Let me know if you need any support here, @OmarAI2003 :)

**Comment by andrewtavis** [comment]
Let's start on #5 first and do a PR for that, @OmarAI2003, and then you're welcome to start with this issue. I think that once this one is done you'll have a lot more of an understanding of what the functionality of the CLI will be, and then you'll be ready to work on #4 :)

For the tests, let's load in the testing files via utils as has been done, and then we can assert that all of the checks fail, as the directory has been set up that there's a problem that will fail each one. From there we can expand it, but that level of testing should be enough for the initial release.

**Comment by andrewtavis** [comment]
I think we're good to close this one and move on to the CLI functionality, @OmarAI2003 😊 From here we can open a new issue for the functionality tests once we know that things are functioning the way we expect them to with the test frontend :)

Thanks for the great step here for `i18n-check` :)
